index.html

```diff
@@ -939,7 +1099,13 @@ <h3>Role Mapping Table</h3>
 								</td>
 								<td>Expose <p><code>ROLE_FOOTER</code> and object attribute
 											<code>xml-roles:doc-pagefooter</code>. </p></td>
-								<td>AXRole: <code>AXGroup</code><br /> AXSubrole: <code>&lt;nil&gt;</code>
+								<td>
+									<ul>
+										<li>AXRole: <code>AXGroup</code></li>
+										<li>AXSubrole: <code>&lt;nil&gt;</code></li>
```

Comment:
TBD… Subrole.

Reply:
Confirming as-is. Use splitter role with additional "footer" description.

Reply:
agreed, and looks like this has been added!